src/pointfree/index.js

```diff
@@ -18,6 +18,8 @@ module.exports = {
   fold: require('./fold'),
   foldMap: require('./foldMap'),
   head: require('./head'),
+  init: require('./init'),
+  last: require('./last'),
```

Comment:
These need to be added to the index.spec.js for the pointfree index. 
You'll also need to add these to the tests for the root `src/index.spec.js`